ko/news/_posts/2024-05-30-datadog-oss-program.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: news_post
+title: "Datadog가 ruby-lang.org의 OSS 커뮤니티 지원을 제공합니다"
+author: "hsbt"
+translator: "shia"
+date: 2024-05-30 00:00:00 +0000
+lang: ko
+---
+
+우리는 루비의 공식 웹사이트인 ruby-lang.org가 [Datadog OSS 커뮤니티 지원](https://opensource.datadoghq.com/projects/oss-program/)을 통한 Datadog 모니터링을 채택했다는 것을 알려드립니다.
+
+Datadog은 루비 사용자를 위해 실시간으로 사이트의 성능과 가용성을 효과적으로 모니터링할 수 있도록 도와줍니다. Datadog 사용의 주요 이점은 다음과 같습니다.
+
+## CDN 가시성
+
+Fastly가 제공하는 cache.ruby-lang.org는 루비 생태계의 가장 중요한 인프라입니다. Datadog를 통해 콘텐츠 전송 네트워크(CDN) 성능을 감시할 수 있습니다. 캐시 커버리지, 에러율을 추적하여 사용자 경험을 향상시킵니다.
+
+## 통합된 데이터 시각화
+
+ruby-lang.org에는 다양한 웹 서비스가 있습니다. Datadog는 로그 분석 데이터를 메트릭 및 애플리케이션 성능 모니터링(APM) 데이터와 함께 동일한 대시보드에서 시각화할 수 있습니다.
```

Comment:
How about this? I tried to make the sentence more natural.

```suggestion
ruby-lang.org에는 다양한 웹 서비스가 있습니다. Datadog는 로그 분석 데이터를 시각화할 때 메트릭 및 애플리케이션 성능 모니터링(APM) 데이터도 같은 대시보드에 함께 보여줄 수 있게 해줍니다.
```

Reply:
much better 👍  fixed 163ac9e